WPFHospitalEditorUnitTests/FindMapObjectsTest.cs

```diff
@@ -0,0 +1,61 @@
+using System;
+using Xunit;
+using WPFHospitalEditor.MapObjectModel;
+using System.Collections.Generic;
+using WPFHospitalEditor;
+using Moq;
+using WPFHospitalEditor.Repository;
+using WPFHospitalEditor.Service;
+using Shouldly;
+using System.Windows;
+using WPFHospitalEditor.Service.Interface;
+
+namespace WPFHospitalEditorUnitTests
+{
+    public class FindMapObjectsTest
+    {
+        [StaFact]
+        public void find_existing_object()
+        {
+            var mapObjects = createMapObjectList();          
+            MapObjectService mapObjectService = new MapObjectService();
+            mapObjectService.iMapObjectRepository = createStubRepository();
+
+            MapObject mapObject = mapObjectService.findMapObjectById(13, mapObjects);
+
+            mapObject.ShouldBeNull();
+        }
+        
+        [StaFact]
+        public void find_non_existing_object()
+        {
+            var mapObjects = createMapObjectList();
+            MapObjectService mapObjectService = new MapObjectService();
+            mapObjectService.iMapObjectRepository = createStubRepository();
+
+            MapObject mapObject = mapObjectService.findMapObjectById(12, mapObjects);
+
+            mapObject.ShouldNotBeNull();
+        }
```

Comment:
Same problem as in the previous test, you should change the name of these two tests.

Reply:
Fixed.